### ISSUE TITLE
how to handle overfitting problem?

### ISSUE THREAD

**Issue by xy-whu** [emadeldeen24/TS-TCC#29]
I have tried the TS-TCC to build a classifier for EEG dataset.
However, the training and validation accuracy are greater 90%, but the test accuracy is only 44%.

So would you like to give me some advice or suggestions ?

Thanks!

**Comment by emadeldeen24** [comment]
Can you please clarify more about the experimental settings and the backbone model you used? TS-TCC is meant for self-supervised training. Do you mean that the overfitting happens in the fine-tuning phase or direct supervised training?

**Comment by emadeldeen24** [comment]
Will close due to inactivity
